misc/docker/README.md

```diff
@@ -50,9 +50,4 @@ docker run -d --network=host -p 5001:5001/tcp --name runner-service runner-servi
 docker run -d --network=host -p 5001:5001/tcp -v /usr/share/ansible-runner-service:/usr/share/ansible-runner-service -v /etc/ansible-runner-service:/etc/ansible-runner-service --name runner-service runner-service
```

Comment:
I have seen that the container (with persistence) does not start even with the "env" folder created. 
This happens if SELINUX=enforcing (/etc/sysconfig/selinux)
Is needed to change this setting to "permissive/disabled" in order to work.
Another workaround is to "disable" temporally with:
# sudo setenforce 0

Reply:
@Daniel-Pivonka @jmolmo Please review the doc and ssh key handling changes added today - I'd like to get this merged in the next couple of days.

Reply:
It should work perfect now!!  ( and i have learn some things!!)